client/src/components/main.css

```diff
@@ -1,18 +1,23 @@
+@font-face {
+	font-family: RobotoMono-Light;
+	src: url(../fonts/RobotoMono-Light.ttf);
```

Comment:
Nit: use a separate service to store your fonts, it's insecure to leave ttfs in a public github repo. Since this font's free it doesn't matter, but just to keep in mind!